Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessDataTab.java

```diff
@@ -70,12 +70,14 @@ public void setDocType(String docType) {
      */
     public String getDocType() {
         if (Objects.isNull(docType)) {
-            String monograph = (String) getAllDoctypes()
-                    .stream()
-                    .filter(typ -> typ.getValue().equals("Monograph"))
-                    .findFirst()
-                    .get().getValue();
-            setDocType(Objects.isNull(monograph) || monograph.isEmpty() ? (String) getAllDoctypes().get(0).getValue() : monograph);
+            List<SelectItem> allDocTypes = getAllDoctypes();
+            for (SelectItem item : allDocTypes) {
+                if ("Monograph".equals(item.getValue())) {
```

Comment:
Should list of supported document / media types (for now only `Monograph`) be hard coded inside the application? This was a hard way to remove this restriction in Kitodo.Production 2.x and adding such kind of restriction should in my humble opinion be avoided.

Reply:
No, such a feature has not been requested so far.

The current state is that each division can be the type of root element. It was discussed that we might later add a possibility that these types can be restricted. In the context, one could also implement the selection of a standard type.